collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I also didn't use it before but many times watching some tutorials I have run into something like:
if you need nested layouts to achieve something it's more performant to use ConstraintLayout instead so why not.

Reply:
Fixed.

Reply:
Yeah it definitely seems like they're concentrating on them over `RelativeLayout`. I'll try and remember to use `ConstraintLayout` instead of `RelativeLayout` but definitely tell me off if I forget!